pages/content_packs.rst

```diff
@@ -0,0 +1,101 @@
+
+How do I create a Content Pack?
+===============================
+
+1. Navigate to **System / Content Pack**.
+2. Click on **Create a Content Pack** on the upper right side of the page.
+   You will now see the wizard for a content pack creation. On the left side
+   of that page you can see the navigation of the wizard. There are 3 steps to
+   content pack creation: **Content Selection**, **Parameters** and **Preview**.
+   In the middle part of the page is the form of each wizard step. And on the
+   right side you can see the summary of your content pack.
+
+.. image:: /images/content_pack_selection.png
+
+3. Fill out the general information of the content pack.
+4. In **Content Selection** you can select the configuration you want to
+   include in your content pack. Necessary dependencies will be included
+   automatically.
+
+.. Warning:: The dependency resolution of pipeline rules does not work yet.
+             Grok patterns and lookup tables of pipeline rules must be add
+             manually to the content pack.
+
+5. Click on **Next** or **Parameter** to go the Parameter Page.
+   Parameter are placeholder which will be filled out during installation
+   of a content pack. That way parts of the configuration can be adjusted
+   fitting to the need of the user.
+   To create a parameter click on **Create Parameter**. In the opening modal
+   you can specify the name, type and default value of the parameter. We
+   support four types of configuration values: String, Integer, Double
+   and Boolean.
+
+.. image:: /images/content_pack_parameter.png
+
+6. After creating a parameter you can assign it to a configuration
+   key. For that press **Edit** on one of the previously selected
+   configuration under **Entity List**.
+7. Press **Next** or **Preview** to complete the creation.
+   On the preview page you get a summarize of your content pack. Please
+   take a close look if all needed configurations are included.
+8. To finish the creation click on **Create** or **Create and Download**.
+
+Installing a content pack
+=========================
+
+To install the newest version of a content pack, navigate to
+**System / Content Pack**.
+Here you see the list of uploaded and created content packs.
+By clicking **Install** on the desired content pack, a modal will open
+which will ask for a **Install comment** and the values of the parameters.
+It also shows the list of configurations that will be installed on the
+system. Press on **Install** to complete the installation.
+
+.. note:: When installing a configuration of a type where the title must be
+          unique (e.g Lookup Tables) the configuration will be only installed
+          once, even when the configuration differs.
+
+Uninstalling a content pack
+===========================
+
+Navigate to **System / Content Pack** and click on the name of the content
+pack that should be uninstalled.
+You see now the page of the uploaded content pack.
```

Comment:
Try to avoid personal pronouns like You. Instead maybe, "The next page will display details of the uploaded content."